src/CONST.ts

```diff
@@ -1213,7 +1213,7 @@ const CONST = {
                 OUTDATED_BANK_ACCOUNT: 'OUTDATEDBANKACCOUNT', // OldDot Action
                 REIMBURSED: 'REIMBURSED',
                 REIMBURSEMENT_ACH_BOUNCE: 'REIMBURSEMENTACHBOUNCE', // OldDot Action
-                REIMBURSEMENT_ACH_CANCELLED: 'REIMBURSEMENTACHCANCELLED', // OldDot Action
+                REIMBURSEMENT_ACH_CANCELLED: 'REIMBURSEMENTACHCANCELED', // OldDot Action
```

Comment:
It seems that both OldDot and Auth chose the american spelling for this. It needs to match.

Reply:
Nice, thanks for this. 🇺🇸 all the way. Let's also update the name of the constant.